downstream/modules/platform/proc-upgrade-controller-hub-eda-unified-ui.adoc

```diff
@@ -5,7 +5,10 @@
 [id="upgrade-controller-hub-eda-unified-ui_{context}"]
 = Automation controller and automation hub 2.4 and EDA 2.5 with unified UI upgrades
 
-{PlatformNameShort} 2.5 supports upgrades from legacy 2.4 environments as well as the current version of {EDAName} to {PlatformNameShort} 2.5 with a unified UI using the RPM install method only. Combining install methods (OCP, RPM, Containerized) within such a topology is not supported by {PlatformNameShort}.
+{PlatformNameShort} 2.5 supports upgrades from {PlatformNameShort} 2.4 environments for all components, with the exception of EDA. You can also configure a mixed environment with {EDAName} from 2.5 connected to a legacy 2.4 cluster. Combining install methods (OCP, RPM, Containerized) within such a topology is not supported by {PlatformNameShort}.
```

Comment:
EDA is not an approved acronym. Changing to {EDAName} attribute in a separate PR.